openupgradelib/openupgrade.py

```diff
@@ -564,6 +564,9 @@ def update_module_names(cr, namespec):
         query = ("UPDATE ir_module_module_dependency SET name = %s "
                  "WHERE name = %s")
         logged_query(cr, query, (new_name, old_name))
+        query = ("UPDATE ir_translation SET module = %s "
```

Comment:
It's an update. Read again the initial comment: `Although not a blocking point, as translations are re-loaded on module update, it's good for avoiding garbage records and preserve possible custom labels.`